docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -74,6 +74,22 @@ In general, Controllers use a Pod Template that you provide to create the Pods f
 Pod templates are pod specifications which are included in other objects, such as
 [Replication Controllers](/docs/concepts/workloads/controllers/replicationcontroller/), [Jobs](/docs/concepts/jobs/run-to-completion-finite-workloads/), and
 [DaemonSets](/docs/concepts/workloads/controllers/daemonset/).  Controllers use Pod Templates to make actual pods.
+The sample below is a simple Pod template which contains a container that prints
+a message.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp-pod
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: myapp-container
+    image: busybox
+    command: ['sh', '-c', 'echo Hello Kubernetes! && sleep 3600']
+```
 
```

Comment:
Suggestion: Call this a Pod manifest instead of a Pod template. Otherwise, readers could confuse this with [PodTemplate](https://kubernetes.io/docs/api-reference/v1.8/#podtemplate-v1-core).

Reply:
+1